lib/sqlbuilder/builder_test.go

```diff
@@ -349,11 +349,16 @@ func TestSelect(t *testing.T) {
 	)
 
 	assert.Equal(
-		`SELECT * FROM "artist" LIMIT -1 OFFSET 5`,
```

Comment:
This is a good point. The need here is we need to be able to remove a previous limit setting somehow. Giving a negative number is often used to signal stuff like this. We can probably use a constant like:

```
Limit(db.NoLimit).Offset(db.NoOffset)
```

and reject any other negative values with an error. I'll go with this change for now because we need it but we can discuss about a different solution for the Limit/Offset case later.